spec/presenters/hyku/file_set_presenter_spec.rb

```diff
@@ -5,22 +5,33 @@
   let(:request) { double(base_url: 'http://test.host') }
   let(:presenter) { described_class.new(solr_document, nil, request) }
   let(:id) { CGI.escape(file_set.original_file.id) }
-  before do
-    Hydra::Works::AddFileToFileSet.call(file_set,
-                                        fixture_file('images/world.png'),
-                                        :original_file)
-  end
 
   describe "display_image" do
-    subject { presenter.display_image }
-    it "creates a display image" do
-      expect(subject).to be_instance_of IIIFManifest::DisplayImage
-      expect(subject.url).to eq "http://test.host/images/#{id}/full/600,/0/default.jpg"
+    context "when the file is not an image" do
+      subject { presenter.display_image }
+      before do
+        Hydra::Works::AddFileToFileSet.call(file_set,
+                                            fixture_file('csv/sample.csv'), :original_file)
+      end
+      it { is_expected.to be_nil }
+    end
+    context "when the file is an image" do
+      subject { presenter.display_image }
+      before do
+        Hydra::Works::AddFileToFileSet.call(file_set,
+                                            fixture_file('images/world.png'), :original_file)
+      end
+      it { is_expected.to be_instance_of IIIFManifest::DisplayImage }
```

Comment:
Would you mind combining these two back to a single it block?  This way is easier to read, but the combined way is a bit faster and we are concerned about how long the build takes to run.